_posts/2023-11-30-indico-3-3-news.md

```diff
@@ -0,0 +1,101 @@
+---
+layout:     post
+title:      "Indico 3.3"
+date:       2023-12-05 10:00
+author:     "Indico Team"
+categories: indico release
+---
+
+Today, we are introducing Indico 3.3! This upcoming release brings a lot of new features and improvements, including a **new Check-in app**, **accessibility improvements (a11y)**, **receipt generation** and a whole lot more.
```

Comment:
The name we're using in the UI for the receipts feature right now is "document regeneration", although I'm not totally opposed to calling it "receipts generation" for now since that's the main use-case I guess...

Reply:
Receipts and attendance certificates. So +1 on "Document generation"